cmd/bosun/conf/conf.go

```diff
@@ -1080,6 +1082,8 @@ func (c *Conf) loadNotification(s *parse.SectionNode) {
 			n.Get = get
 		case "print":
 			n.Print = true
+		case "contentType":
+			n.ContentType = v
```

Comment:
We could potentially do a `mime.ParseMediaType(v)` for validation here, but I am not sure it is necessary.